src/injection.yaml

```diff
@@ -126,7 +126,7 @@ repository:
       1: { name: punctuation.definition.interpreted.sphinx }
```

Comment:
I believe it would be more consistent to also introduce `.start` sub token here to match the `.end` one below.

Reply:
Agreed